_data/scout-cli/docker_scout_quickview.yaml

```diff
@@ -0,0 +1,91 @@
+command: docker scout quickview
+short: Quick overview of an image
+long: |-
+    The `docker scout quickview` command displays a quick overview of an image.
+    It displays a summary of the vulnerabilities in the image and the one specifically in the base image.
```

Comment:
@eunomie What does "the one specifically" mean? Maybe is it the vulnerabilities in the image that come from the base image?